lib/charts/src/cartesian_area_impl.dart

```diff
@@ -423,7 +423,8 @@ class DefaultCartesianAreaImpl implements CartesianArea {
     config.dimensions.take(dimensionAxesCount).forEach((int column) {
       var axis = _getDimensionAxis(column),
           sampleColumnSpec = data.columns.elementAt(column),
-          values = data.rows.map((row) => row.elementAt(column));
+          values = data.rows.map((row) => row.elementAt(column))
```

Comment:
map<Comparable> instead?

Reply:
I think you're type annotations got swallowed by GitHub Markdown :grinning: but I think I know what you mean.

Reply:
updated. :)

Reply:
Analyzer doesn't like that:

> error • The return type 'dynamic' isn't a 'Comparable', as defined by anonymous closure at lib/charts/src/cartesian_area_impl.dart:426:55 • return_of_invalid_type_from_closure

unless I use `as Comparable` inside as well.

Reply:
Ah, implicit casts must be off, good!  Putting the cast inside of the lambda is better.  The cast on the outside probably will fail strong mode runtime checks.

Reply:
Done!